CMakeLists.txt

```diff
@@ -137,6 +148,7 @@ if(GIT_FOUND AND PYTHONINTERP_FOUND)
   endif()
 else()
   set(GIT_FIRMWARE_VERSION_STRING "v0.0.0")
+  set(GIT_FIRMWARE_BTC_VERSION_STRING "v0.0.0")
```

Comment:
For consistency, I'd prefer `BTCONLY` everywhere instead of `BTC`.

Reply:
for consistency I named it `BTC_ONLY`.